tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -1047,7 +1047,11 @@ public class ExpressionParserTests
             Test("lastIndexOf(newGuid(), '-')", 23),
             Test("lastIndexOf(hello, '-')", -1),
             Test("lastIndexOf(nullObj, '-')", -1),
+#if NET5_0_OR_GREATER
```

Comment:
Can you provide more context on this? lastIndexOf should behave the same regardless of platform, I "think" the tests should pass and we may need to fix this somewhere else.

Reply:
@EricDahlvang found this issue https://github.com/dotnet/docs/issues/28218, see if this helps to implement a workaround

Reply:
Can we document this with links to the dotnet bug in code so other can understand why this was done?